tests/test_mtu.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
+from datetime import datetime
+
+
+@pytest.mark.parametrize("mtu", [1514,9114])
+def test_mtu(ansible_adhoc, testbed, duthost, ptfhost, mtu):
+
+    hostname = testbed['dut']
+    testbed_type = testbed['topo']['name']
+
+    props= testbed['topo']['properties']['configuration_properties']['spine']
+    props_tor = testbed['topo']['properties']['configuration_properties']['tor']
+
+    ans_host = AnsibleHost(ansible_adhoc, hostname)
```

Comment:
These variables are not needed. Suggest to remove them: `hostname`, `props`, `props_tor`, `ans_host`.

Regarding `ans_host`, `duthost` is already a good replacement for it.

Reply:
Modification done as per the comments